### PR TITLE
Refactor Runtime Provisioner and Connector charts

### PR DESCRIPTION
**Pull request by franpog859**
**Description**

Changes proposed in this pull request:

- Refactor Runtime Provisioner and Connector charts

**Related issue(s)**

See the issue #1388

### CODE REVIEW

File: chart/compass/charts/provisioner/values.yaml [hunk 1]
```diff
@@ -3,8 +3,6 @@ deployment:
   image:
     pullPolicy: IfNotPresent
   
-  resources: {}
```
Comment (Szymongib) [inline chart/compass/charts/provisioner/values.yaml:6]: I think we are using it in the Deployment https://github.com/kyma-incubator/compass/pull/1397/files#diff-4b7a93aa1a609a4734f843e3a6f98875L41-L42 (line 42)

Comment (franpog859) [reply chart/compass/charts/provisioner/values.yaml:6]: Sure! My fault. It was broken anyway - it should be `.Values.deployment.resources`. I fixed it already. Same situation in the Connector chart

File: chart/compass/charts/connector/templates/certs-setup-role-binding.yaml [hunk 2]
```diff
@@ -1,12 +1,15 @@
+{{- if .Values.certsSetupJob.enabled }}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ .Chart.Name }}-certs-setup-job
   namespace: {{ .Release.Namespace }}
   labels:
     app: {{ .Chart.Name }}-certs-setup-job
-    release: {{ .Release.Name }}
```
[the remark this answers is not in the document]
Comment (franpog859) [reply chart/compass/charts/connector/templates/certs-setup-role-binding.yaml:8]: True, it'll make our charts more consistent. I won't remove the `release` label then

File: chart/compass/charts/connector/templates/deployment.yaml [hunk 3]
```diff
@@ -5,19 +5,25 @@ metadata:
...
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
 spec:
   selector:
     matchLabels:
       app: {{ .Chart.Name }}
-      release: {{ .Release.Name }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
```
[the remark this answers is not in the document]
Comment (franpog859) [reply chart/compass/charts/connector/templates/deployment.yaml:16]: Yeah, it seems to be. That's disappointing tho ;<

File: chart/compass/charts/provisioner/templates/deployment.yaml [hunk 4]
```diff
@@ -5,20 +5,26 @@ metadata:
...
+    app.kubernetes.io/name: {{ template "name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
 spec:
   replicas: {{ .Values.deployment.replicaCount }}
   selector:
     matchLabels:
       app: {{ .Chart.Name }}
-      release: {{ .Release.Name }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
```
[the remark this answers is not in the document]
Comment (franpog859) [reply chart/compass/charts/provisioner/templates/deployment.yaml]: Done!

**Pull request merged**